Assets/Scripts/MainMenuManager.cs

```diff
@@ -77,4 +73,9 @@ private int menuSelection(string[] menuItems, int selectedItem, string direction
 
         return selectedItem;
     }
+
+    public void NewGame()
+    {
+        SceneManager.LoadScene("Level_01");
```

Comment:
There is an ExtendedSceneManager that will take the Enum value for SceneNames now.